stories/core/Table.stories.tsx

```diff
@@ -2292,7 +2295,152 @@ VirtualizedSubRows.argTypes = {
   data: { control: { disable: true } },
 };
 
+export const ColumnManager: Story<Partial<TableProps>> = (args) => {
```

Comment:
Currently, test image does not really make sense... Maybe better take a screenshot of opened menu too?
YOu can find some examples of custom actions in `Select` stories, for example.. Just of course, you need to add such test for individual story (yours), not all the table stories.

Reply:
![image](https://user-images.githubusercontent.com/78564221/160302700-d2cdfc06-072c-4c79-91a5-e05bc832323f.png)

This is the current opened test image I have. Is it okay if it's cut off, or should I put less columns in the story so that they all fit in the screenshot?

Reply:
So 2 options here.
- Just add more rows
- Add wrapper element with some padding so everything is visible in screenshot (you can find examples in our stories with `decorators`

Reply:
Added extra rows to Column Manager story

Reply:
I think you still have one old image left from previous visuals (before you added logic to open and take screenshot).

Reply:
Test image removed